documentation/book/proc-adding-users-the-strimzi-admin-role.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
```

Comment:
On a general note, should this procedure include a reference to the new YAML file?